app/dbmodels.py

```diff
@@ -42,9 +39,58 @@ class Meta(BaseMeta):
     settings: pydantic.Json = ormar.JSON()
 
 
+class Conversations(ormar.Model):
+    class Meta(BaseMeta):
+        tablename = "conversations"
+    id: int = ormar.Integer(primary_key=True)
+    type: str = ormar.String(max_length=100)
+    settings: pydantic.Json = ormar.JSON()
+    owner_id: int = ormar.Integer()
```

Comment:
```suggestion
    owner_id: User =ormar.ForeignKey(User)
```
Чтобы сразу связать этот столбец с объектом пользователя, лучше объявить его как внешний ключ. Вот таким образом.